modules/teadsBidAdapter.js

```diff
@@ -129,7 +129,9 @@ function buildRequestObject(bid) {
 
 function getSizes(bid) {
   let sizes;
-  if (isVideoBid(bid)) {
+  if (isHybridBid(bid)) {
+    sizes = concatHybridBidSizes(bid);
+  } else if (isVideoBid(bid)) {
```

Comment:
Have a couple comments / suggestions:

With this current implementation a multi-format ad unit will add `video.sizes` and `video.playerSize`.

But if it is _just_ a video mediaType, then it picks `video.playerSize` OR `video.sizes`. Not sure if it is better to remain consistent and just concatenate in this scenario as well?

Or maybe in `concatHybridBidSizes` add the logic to choose `video.playerSize` OR `video.sizes`.

Also, instead of checking the `isHybridBid` or `isVideoBid` => `else`, does it make sense to just use the `concatHybridBidSizes` and not do any mediaTypes checking?

Since you are using `utils.deepAccess` to get the mediaTypes Sizes, then if the mediaType is not present, it will do what is needed anyways right?

so `getSizes` could just be something like:

```javascript
function getSizes(bid) {
   return utils.parseSizesInput(concatHybridBidSizes(bid));
}
```

And if needed add the `video.playerSize` || `video.sizes` logic, if that is the intended implementation.

Reply:
CC @muuki88 for some input as well

Reply:
Exact ! I did the change, simpler and cleaner :)

Reply:
Makes sense :smiley: